docker-for-mac/index.md

```diff
@@ -462,6 +462,28 @@ ln -s $etc/docker.zsh-completion /usr/local/share/zsh/site-functions/_docker
 ln -s $etc/docker-compose.zsh-completion /usr/local/share/zsh/site-functions/_docker-compose
 ```
 
+### Fish-Shell
+
+Fish-shell also supports tab completion [completion
+system](https://fishshell.com/docs/current/#tab-completion){:target="_blank"
+class="_"}. To activate completion for Docker commands,
+these files need to be copied or symlinked to your Fish-shell `completions/`
+directory.
+
+Create at first the `completions` directory if not exist:
+
+```bash
+mkdir ~/.config/fish/completions
+```
+
+Now add fish completions from docker.
+
+```bash
+set etc /Applications/Docker.app/Contents/Resources/etc
+ln -s $etc/docker.fish-completion ~/.config/fish/completions/docker.fish
+ln -s $etc/docker-compose.fish-completion ~/.config/fish/completions/docker-compose.fish
```

Comment:
Bit on the fence if we should use the intermediate variable (as it will only be used once), or just inline the path in the commands;

```suggestion
ln -s /Applications/Docker.app/Contents/Resources/etc/docker.fish-completion ~/.config/fish/completions/docker.fish
ln -s /Applications/Docker.app/Contents/Resources/etc/docker-compose.fish-completion ~/.config/fish/completions/docker-compose.fish
```

Also thinking if we should add the `h` (don't follow symlink if the target is a symlink) and `i` (prompt for overwriting target if it exists) options; WDYT?

```suggestion
ln -shi /Applications/Docker.app/Contents/Resources/etc/docker.fish-completion ~/.config/fish/completions/docker.fish
ln -shi /Applications/Docker.app/Contents/Resources/etc/docker-compose.fish-completion ~/.config/fish/completions/docker-compose.fish
```